schema/artwork/index.js

```diff
@@ -32,6 +34,17 @@ const ArtworkType = new GraphQLObjectType({
       _id: {
         type: GraphQLString,
       },
+      to_s: {
+        type: GraphQLString,
+        resolve: ({ artist, title, date, partner }) => {
+          return _.compact([
+            (artist && artist.name),
+            (title && `‘${title}’`),
+            date,
+            (partner && partner.name),
+          ]).join(', ');
+        },
+      },
```

Comment:
This is debateable: trying to co-opt the Rails convention. I imagine we could write `to_s` fields for most models instead of `displayFoobar`